docs/how-to/create-a-router/configure-traffic-rules.md

```diff
@@ -31,5 +47,5 @@ Provided values are case-sensitive.
 You should also select one or more routes from the drop-down list that would be activated if the rule is triggered. A route can be attached to zero or more traffic rules.
 
 * If a route is attached to some traffic rule, then Turing will only send request to this route if the request meets this rule's conditions.
-* If a route is not attached to any of the traffic rules, then Turing will call this route with every incoming request.
+* If a route is not attached to any of the traffic rules (i.e no traffic rules specified), then Turing will call this route with every incoming request.
```

Comment:
Eh this is no longer the case no?

Reply:
Yep, unclear docs here - I've removed it.